srcpkgs/lazyorg/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'lazyorg'
+pkgname=lazyorg
+version=1.1.1
+revision=1
+build_style=go
+go_import_path=github.com/HubertBel/lazyorg
+go_package="./cmd/lazyorg"
+short_desc="Simple terminal-based calendar and note-taking application"
+maintainer="Xavier Fortier <xavier.fortier@proton.me>"
+license="MIT"
+homepage="https://github.com/HubertBel/lazyorg"
+distfiles="${homepage}/archive/v${version}.tar.gz"
```

Comment:
Other templates generally don't use `${homepage}` here but the full url instead, technically makes no difference but it probably makes sense to follow what other packages do to keep things consistent

Reply:
I have seen both, so I thought both were ok, but when I looked into the manual, the full url is preferred.